src/control/AxesController.ts

```diff
@@ -368,7 +368,7 @@ class AxesController {
   };
 
   private _onAxesChange = () => {
-    this._dragged = true;
+    this._dragged = !!this._panInput?.isEnabled() && true;
```

Comment:
How about checking it with properties like `this._state.holding` or `isTrusted` property of the Axes event?
Adding more inputs like WheelInput can break this.

Reply:
Since we are blocking the click event through the `preventClickOnDrag` option,
I don't think there's any need to prevent click event when we move Panel with Wheel or MoveKey etc.

Reply:
That makes sense 👍

Reply:
Aside from that, `&& true` was removed because it looked unnecessary.